pkg/signal/testfiles/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"os"
+	"syscall"
+	"time"
+
+	"github.com/docker/docker/pkg/signal"
+)
+
+func main() {
+	sigmap := map[string]os.Signal{
+		"TERM": syscall.SIGTERM,
+		"QUIT": syscall.SIGQUIT,
+		"INT":  os.Interrupt,
+	}
+
+	defer time.Sleep(10 * time.Millisecond)
```

Comment:
Ok, why defer instead of just sleeping at the end of the function? I think that would be more obvious.

Also, isn't that what the `select {}` at the end is for? Can we remove that if we're going to sleep?